visualization/app/codeCharta/util/treeMapHelper.ts

```diff
@@ -28,6 +28,39 @@ export class TreeMapHelper {
 		return geomMap
 	}
 
+	public static buildRootFolderForFixedFolders(map: CodeMapNode, heightScale: number, state: State, isDeltaState: boolean): Node {
+		const flattened: boolean = this.isNodeToBeFlat(map, state)
+		const height = 2
+
+		return {
+			name: map.name,
+			id: 0,
+			width: 100,
+			height: 2,
+			length: 100,
+			depth: 0,
+			x0: 0,
+			z0: 0,
+			y0: 0,
+			isLeaf: false,
+			attributes: map.attributes,
+			edgeAttributes: map.edgeAttributes,
+			deltas: map.deltas,
```

Comment:
They're not copied, they're just passed by reference.

Reply:
Sorry, I guess I could have worded this better :)

What I meant was if all properties of the `map` are currently shallowly copied to the new object such as `{ attributes: map.attributes }`. If that's the case, it would be possible to do: `{ foo: 5, bar: 'value', ...map }`. If not _all_ properties from `map` get copied, that's not the way to go but I thought I'll ask.

Reply:
I still can't follow yet. Let's talk about this tomorrow. :)